crates/net/network/README.md

```diff
@@ -1 +1,5 @@
-# RETH network implementation
\ No newline at end of file
+# RETH network implementation
+
+Supports both devp2p peer discovery version 4 and 5. The default is Discv4. Discv5 
+is available as a feature. Behind the scenes, Discv5 runs Discv4 too, for 
```

Comment:
so they're no longer independent of each other?

do we want this?

Reply:
atm on mainnet ethereum no, since other client impls are running on discv4, but in other evm chains it's a good idea to allow running with only discv5